src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -2080,8 +2093,9 @@ export default class Aircraft {
 
                     prop.game.score.abort.landing += 1;
                 }
-            } else if (this.altitude >= 300) {
-                this.target.heading = this.fms.currentWaypoint().heading;
+            }
+            else if (this.altitude >= 300) {
```

Comment:
make sure an else if is on the same line as the closing brace for the previous conditional.